TestPlaybooks/playbook-Slack_TestPlaybook.yml

```diff
@@ -323,12 +329,180 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+  "10":
```

Comment:
can we add the invite user, kick user, rename channel and get user commands to the test playbook ?